transports/file/src/main/java/org/mule/transport/file/FileMessageReceiver.java

```diff
@@ -60,10 +62,13 @@
 
 public class FileMessageReceiver extends AbstractPollingMessageReceiver
 {
+
     public static final String COMPARATOR_CLASS_NAME_PROPERTY = "comparator";
     public static final String COMPARATOR_REVERSE_ORDER_PROPERTY = "reverseOrder";
     public static final String MULE_TRANSPORT_FILE_SINGLEPOLLINSTANCE = "mule.transport.file.singlepollinstance";
+    public static final String NOT_PROCESS_EMPTY_FILES_PROPERTY = SYSTEM_PROPERTY_PREFIX + "transport.file.notProcessEmptyFiles";
 
+    private final Boolean NOT_PROCESS_EMPTY_FILES = getBoolean(NOT_PROCESS_EMPTY_FILES_PROPERTY);
```

Comment:
Make the name positive (like SKIP_EMPTY_FILES, IGNORE_EMPTY_FILES or PROCESS_EMPTY_FILES)